code/game/objects/items/storage/storage.dm

```diff
@@ -447,6 +450,12 @@
 	update_icon()
 	return TRUE
 
+/// Delete everything that's inside the storage
+/obj/item/storage/proc/delete_content()
+	for(var/obj/item/I in contents)
```

Comment:
delete_contents please to emphasize the list this is iterating over
Also missing AS?